app/controllers/users_controller.rb

```diff
@@ -22,6 +22,10 @@ def create
     end
   end
 
+  def edit
+    @user = User.find_by id: params[:id]
```

Comment:
Lặp code lại phần tìm user ở một số action rồi nè, tách riêng ra method xử lý rồi dùng before_action để gọi nhé
Với xử lý thêm trường hợp @user bị nil nữa
```suggestion
    @user = User.find_by id: params[:id]
     return if @user
     
     flash ...
     redirect_to ...
```

Reply:
dạ